src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -7365,6 +7365,9 @@ def create_new_cluster_with_monitoring_aad_auth(self, resource_group, resource_g
             ])
              self.cmd(get_cmd, checks=[
                 self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.namespaces[0]', f'kube-system')
+            ])
+             self.cmd(get_cmd, checks=[
+                self.check('properties.dataSources.extensions[0].extensionSettings.dataCollectionSettings.enableContainerLogV2', f'true')
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=75185750&view=results) (case `test_aks_create_with_monitoring_aad_auth_msi_with_datacollectionsettings`) to validate the change.